whatshap/cli/__init__.py

```diff
@@ -188,7 +188,13 @@ def read(self, chromosome, variants, sample, *, read_vcf=True, regions=None):
         return readset, vcf_source_ids
 
 
-def log_memory_usage():
+def log_memory_usage(multiprocessing=False):
```

Comment:
That parameter name seems a bit overspecific. Can you perhaps call it `include_children`?